ca_gazebo/CMakeLists.txt

```diff
@@ -24,16 +24,25 @@ else()
 			ca_msgs
 	)
 
-	link_directories(${GAZEBO_LIBRARY_DIRS})
-  include_directories(include
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
Why did you add this here having [this one](https://github.com/stevendes/create_autonomy/blob/8481c5fd5120ed942570ff5e8b1de2d0c5dc97fe/ca_gazebo/CMakeLists.txt#L5)?